components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -1578,12 +1578,9 @@ - (void)textInputDidChange:(NSNotification *)note {
         [self.characterCounter characterCountForTextInput:self.textInput];
     NSString *announcementString;
     if (!announcementString.length) {
-      announcementString =
-          [NSString stringWithFormat:@"%lu characters remaining",
-                                     charactersForTextInput > self.characterCountMax
-                                         ? 0U
```

Comment:
So we're OK saying, "-3 of 5 characters"?

Reply:
We were previously just repeating "0 characters remaining". The example you provided would be announced as "8 of 5 characters".

Reply:
Thanks, I was misreading the changed code. Should we update FullWidth as well?